docs/docs/guide/05-scavenge/03-scaffolding.md

```diff
@@ -40,8 +43,8 @@ participants.
 ignite scaffold module scavenge --dep bank
 ```
 
-This command creates the `x/scavenge` directory and imports the scavenge module into the blockchain in the `app/app.go`
-directory.
+This command creates the `x/scavenge` directory and imports the scavenge module
+into the blockchain in the `app/app.go` directory.
```

Comment:
I don't think "`app/app.go` directory" makes sense:

- `app/app.go` file
- `app` directory`
- `app/app.go`

maybe one of these is better.